bots and matcher_service fail startup on integration-test

Exceptions are:

```
Uncaught error from thread [ClusterSystem-akka.actor.default-dispatcher-2] shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled for ActorSystem[ClusterSystem]
java.lang.ExceptionInInitializerError
        at won.protocol.vocabulary.WON.<clinit>(WON.java:34)
        at won.matcher.service.nodemanager.service.WonNodeSparqlService.retrieveAllWonNodeInfo(WonNodeSparqlService.java:43)
        at won.matcher.service.nodemanager.actor.WonNodeControllerActor.preStart(WonNodeControllerActor.java:102)
        at akka.actor.Actor$class.aroundPreStart(Actor.scala:489)
        at akka.actor.UntypedActor.aroundPreStart(UntypedActor.scala:95)
        at akka.actor.ActorCell.create(ActorCell.scala:590)
        at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:461)
        at akka.actor.ActorCell.systemInvoke(ActorCell.scala:483)
        at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:282)
        at akka.dispatch.Mailbox.run(Mailbox.scala:223)
        at akka.dispatch.Mailbox.exec(Mailbox.scala:234)
        at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.lang.NullPointerException
        at org.apache.jena.tdb.sys.EnvTDB.processGlobalSystemProperties(EnvTDB.java:33)
        at org.apache.jena.tdb.TDB.init(TDB.java:248)
        at org.apache.jena.tdb.sys.InitTDB.start(InitTDB.java:29)
        at org.apache.jena.system.JenaSystem.lambda$init$1(JenaSystem.java:111)
        at java.util.ArrayList.forEach(ArrayList.java:1249)
        at org.apache.jena.system.JenaSystem.forEach(JenaSystem.java:186)
        at org.apache.jena.system.JenaSystem.forEach(JenaSystem.java:163)
        at org.apache.jena.system.JenaSystem.init(JenaSystem.java:109)
        at org.apache.jena.rdf.model.ModelFactory.<clinit>(ModelFactory.java:49)
        ... 15 more
[ERROR] [03/14/2017 09:44:27.599] [ClusterSystem-akka.actor.default-dispatcher-2] [akka.actor.ActorSystemImpl(ClusterSystem)] Uncaught error from thread [ClusterSystem-akka.actor.default-dispatcher-2] shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled
java.lang.ExceptionInInitializerError
        at won.protocol.vocabulary.WON.<clinit>(WON.java:34)
        at won.matcher.service.nodemanager.service.WonNodeSparqlService.retrieveAllWonNodeInfo(WonNodeSparqlService.java:43)
        at won.matcher.service.nodemanager.actor.WonNodeControllerActor.preStart(WonNodeControllerActor.java:102)
        at akka.actor.Actor$class.aroundPreStart(Actor.scala:489)
        at akka.actor.UntypedActor.aroundPreStart(UntypedActor.scala:95)
        at akka.actor.ActorCell.create(ActorCell.scala:590)
        at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:461)
        at akka.actor.ActorCell.systemInvoke(ActorCell.scala:483)
        at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:282)
        at akka.dispatch.Mailbox.run(Mailbox.scala:223)
        at akka.dispatch.Mailbox.exec(Mailbox.scala:234)
        at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.lang.NullPointerException
        at org.apache.jena.tdb.sys.EnvTDB.processGlobalSystemProperties(EnvTDB.java:33)
        at org.apache.jena.tdb.TDB.init(TDB.java:248)
        at org.apache.jena.tdb.sys.InitTDB.start(InitTDB.java:29)
        at org.apache.jena.system.JenaSystem.lambda$init$1(JenaSystem.java:111)
        at java.util.ArrayList.forEach(ArrayList.java:1249)
        at org.apache.jena.system.JenaSystem.forEach(JenaSystem.java:186)
        at org.apache.jena.system.JenaSystem.forEach(JenaSystem.java:163)
        at org.apache.jena.system.JenaSystem.init(JenaSystem.java:109)
        at org.apache.jena.rdf.model.ModelFactory.<clinit>(ModelFactory.java:49)
        ... 15 more
```
		
		
```
14.3.17 09:48:42.560 [main] WARN  o.s.c.a.AnnotationConfigApplicationContext - Exception encountered during context initialization - cancelling refresh attempt
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'connectionRepository': Invocation of init method failed; nested exception is java.lang.ExceptionInInitializerError
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1568) ~[bots.jar:na]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:540) ~[bots.jar:na]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476) ~[bots.jar:na]
        at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:302) ~[bots.jar:na]
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:229) ~[bots.jar:na]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:298) ~[bots.jar:na]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:193) ~[bots.jar:na]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:706) ~[bots.jar:na]
        at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:757) ~[bots.jar:na]
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:480) ~[bots.jar:na]
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:648) [bots.jar:na]
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:311) [bots.jar:na]
        at won.bot.app.DebugBotApp.main(DebugBotApp.java:32) [bots.jar:na]
Caused by: java.lang.ExceptionInInitializerError: null
        at won.protocol.vocabulary.WONMSG.<clinit>(WONMSG.java:25) ~[bots.jar:na]
        at won.protocol.message.WonMessageType.<clinit>(WonMessageType.java:15) ~[bots.jar:na]
        at java.lang.Class.forName0(Native Method) ~[na:1.8.0_111]
        at java.lang.Class.forName(Class.java:264) ~[na:1.8.0_111]
        at com.sun.proxy.$Proxy74.<clinit>(Unknown Source) ~[na:na]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_111]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_111]
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_111]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_111]
        at java.lang.reflect.Proxy.newProxyInstance(Proxy.java:739) ~[na:1.8.0_111]
        at org.springframework.aop.framework.JdkDynamicAopProxy.getProxy(JdkDynamicAopProxy.java:121) ~[bots.jar:na]
        at org.springframework.aop.framework.ProxyFactory.getProxy(ProxyFactory.java:109) ~[bots.jar:na]
        at org.springframework.data.repository.core.support.RepositoryFactorySupport.getRepository(RepositoryFactorySupport.java:192) ~[bots.jar:na]
        at org.springframework.data.repository.core.support.RepositoryFactoryBeanSupport.initAndReturn(RepositoryFactoryBeanSupport.java:239) ~[bots.jar:na]
        at org.springframework.data.repository.core.support.RepositoryFactoryBeanSupport.afterPropertiesSet(RepositoryFactoryBeanSupport.java:225) ~[bots.jar:na]
        at org.springframework.data.jpa.repository.support.JpaRepositoryFactoryBean.afterPropertiesSet(JpaRepositoryFactoryBean.java:92) ~[bots.jar:na]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1627) ~[bots.jar:na]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1564) ~[bots.jar:na]
        ... 12 common frames omitted
Caused by: java.lang.NullPointerException: null
        at org.apache.jena.tdb.sys.EnvTDB.processGlobalSystemProperties(EnvTDB.java:33) ~[bots.jar:na]
        at org.apache.jena.tdb.TDB.init(TDB.java:248) ~[bots.jar:na]
        at org.apache.jena.tdb.sys.InitTDB.start(InitTDB.java:29) ~[bots.jar:na]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1568)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:540)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
        at org.apache.jena.system.JenaSystem.lambda$init$1(JenaSystem.java:111) ~[bots.jar:na]
        at java.util.ArrayList.forEach(ArrayList.java:1249) ~[na:1.8.0_111]
        at org.apache.jena.system.JenaSystem.forEach(JenaSystem.java:186) ~[bots.jar:na]
        at org.apache.jena.system.JenaSystem.forEach(JenaSystem.java:163) ~[bots.jar:na]
        at org.apache.jena.system.JenaSystem.init(JenaSystem.java:109) ~[bots.jar:na]
        at org.apache.jena.rdf.model.ModelFactory.<clinit>(ModelFactory.java:49) ~[bots.jar:na]
        ... 30 common frames omitted
```

--- a/webofneeds/won-bot/pom.xml
+++ b/webofneeds/won-bot/pom.xml
@@ -76,6 +76,7 @@
                                 </filter>
                             </filters>
                             <transformers>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
                                     <resource>reference.conf</resource>
                                 </transformer>

--- a/webofneeds/won-matcher-service/pom.xml
+++ b/webofneeds/won-matcher-service/pom.xml
@@ -120,6 +120,7 @@
                                 </filter>
                             </filters>
                             <transformers>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
                                     <resource>reference.conf</resource>
                                 </transformer>

--- a/webofneeds/won-matcher-solr/pom.xml
+++ b/webofneeds/won-matcher-solr/pom.xml
@@ -122,6 +122,7 @@
                                 </includes>
                             </artifactSet>
                             <transformers>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
                                     <resource>reference.conf</resource>
                                 </transformer>

